### DISCUSSION TITLE
希望增加专栏显示原图的设置

### DISCUSSION THREAD

**Discussion by scythk** [the1812/Bilibili-Evolved#2868] [功能建议 / Ideas]
开启了高分辨率图片插件后 专栏的图片依然比较模糊
希望增加显示原图功能 直接把图片链接里的webp后缀删除
如
https://i0.hdslb.com/bfs/article/0d49da6eb5082e0645a042ffcff1d94e70a601f4.png@1295w_815h_progressive.webp
变成
https://i0.hdslb.com/bfs/article/0d49da6eb5082e0645a042ffcff1d94e70a601f4.png

**Comment by qtqz** [comment]
感觉做成点击图片查看原图更好些
